Add docker compose configuration notes (#2967)
grafana/alloy: docs/sources/reference/components/prometheus/prometheus.exporter.cadvisor.md

}
```

> When exporting to Mimir then make sure to set max_label_names_per_series higher than the default, otherwise Mimir will not except the metrics.

Reviewer: ```suggestion
When you export to Mimir, make sure to set `max_label_names_per_series` higher than the default, otherwise Mimir will not accept the metrics.
```

```yaml
limits:
  max_label_names_per_series: 100
```

## Docker Configuration

> To properly collect container metrics when using Alloy in docker compose, the Alloy container must be run with specific volume mounts and privileges. Here's the required Docker Compose configuration:

Reviewer: ```suggestion
To properly collect container metrics when using {{< param "PRODUCT_NAME" >}} in a Docker container, the {{< param "PRODUCT_NAME" >}} container must be run with specific volume mounts and privileges.
The following example shows a typical Docker configuration that provides the correct volume mounts.
```
